troposphere/codedeploy.py

```diff
@@ -164,6 +173,30 @@ class TriggerConfig(AWSProperty):
     }
 
 
+class Ec2TagGroup(AWSProperty):
+    props = {
+        'Ec2TargetGroup': ([Ec2TagFilters], False)
+    }
+
+
+class Ec2TagSet(AWSProperty):
+    props = {
+        'Ec2TagSet': ([Ec2TagGroup], False)
```

Comment:
This looks like the property is `EC2TagSetList`, which is a list of `EC2TagSetListObject` objects,